packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -437,6 +457,91 @@
           }
         }
       ]
+    },
+    "visibility": {
+      "description": "Controls if this item is visible. Value must be 'Disabled' (valid only for Variable items), 'Hidden', 'Visible', or a {{...}} expression against variables that evaluates to one of those or true (Visible) or false (Hidden).",
+      "oneOf": [
+        {
+          "type": "string",
+          "pattern": "^\\{\\{.+\\}\\}$",
+          "patternErrorMessage": "Value must be 'Disabled', 'Hidden', 'Visible', or a {{...}} expression.",
+          "examples": [
+            "{{Variables.booleanVariable}}",
+            "{{Variables.booleanVariable ? 'Visible' : 'Disabled'}}",
+            "{{Variables.stringVariable == 'Yes' ? 'Visible' : 'Hidden'}}",
+            "{{Variables.numberVariable == 5 ? 'Visible' : 'Hidden'}}",
+            "{{Variables.stringArray contains 'X' ? 'Visible' : 'Hidden'}}",
+            "{{Variables.numberArray contains 5 ? 'Visible' : 'Hidden'}}"
+          ]
+        },
+        {
+          "type": "string",
+          "default": "Visible",
+          "enum": ["Disabled", "Hidden", "Visible"],
+          "enumDescriptions": ["The variable shows as disabled", "The item is hidden.", "The item is visible."]
+        },
+        {
+          "type": "string",
+          "$comment": "json-schema does not support case-insensitive enums, so this simulates it, using a lower-case 1st char to avoid a double schema match.",
+          "pattern": "^(d[Ii][Ss][Aa][Bb][Ll][Ee][Dd]|h[Ii][Dd][Dd][Ee][Nn]|v[Ii][Ss][Ii][Bb][Ll][Ee])$"
+        }
+      ]
+    },
+    "variableItem": {
+      "properties": {
```

Comment:
I think these extracted types need `"type": "object"` in them -- there's a warning now when running the analyticsdx-template-lint unit tests:

> @salesforce/analyticsdx-template-lint: strict mode: missing type "object" for keyword "properties" at "http://schemas.salesforce.com/analytics/templates/layout.json#/definitions/imageItem" (strictTypes)